client/index.ts

```diff
@@ -30,4 +32,6 @@ export {
   LocationTrigger,
   IHeroCreateOptions,
   IConnectionToCoreOptions,
+  ISuperElement,
```

Comment:
Are there other ones of the that should get exported? Like SuperNodeList or anything?

Reply:
I don't know, but I needed these two ;)